CPP/Alternative_positive_negative.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+   * कर्मणये वाधिकारस्ते मां फलेषु कदाचन ।
+   * मां कर्मफलहेतुर्भू: मांते संङगोस्त्वकर्मणि ॥
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+#define bug(...)       __f (#__VA_ARGS__, __VA_ARGS__)
```

Comment:
what does this define bug (...) do? pls comment and add to your code. Line no 9.